### DISCUSSION TITLE
Is `gRPCCertificates-Cpp.bundle` necessary?

### DISCUSSION THREAD

**Discussion by uhooi** [firebase/firebase-ios-sdk#9183] [Q&A] [answered]
`Firebase.zip` contains `Firebase/FirebaseFirestore/Resources/gRPCCertificates-Cpp.bundle`.  
https://github.com/firebase/firebase-ios-sdk/releases/tag/v8.10.0

But, I think it is no longer needed since `6.13.0`.  
https://firebase.google.com/support/release-notes/ios#cloud-firestore_35

Why is `gRPCCertificates-Cpp.bundle` still included in the release?
If it is not needed, please remove it as it is confusing.

---

- Firebase SDK version: 8.10.0

**Comment by paulb777** [comment, marked as answer]
I'll transfer to Issue #9184 and discuss there.
